recipes/ghost/index.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  addBlitzMiddleware,
+  addImport,
+  findModuleExportsExpressions,
+  paths,
+  RecipeBuilder,
+} from "@blitzjs/installer"
+import j from "jscodeshift"
+import {Collection} from "jscodeshift"
+import path from "path"
+
+export default RecipeBuilder()
+  .setName("Ghost")
+  .setDescription("Use Ghost in blitz.")
```

Comment:
Is this change accurate?

```suggestion
  .setDescription("Access your Ghost CMS directly in blitz via the Ghost SDK.")
```

Reply:
Much better wording :)